gravitee-apim-plugin/gravitee-apim-plugin-entrypoint/gravitee-apim-plugin-entrypoint-sse/src/main/resources/plugin.properties

```diff
@@ -0,0 +1,6 @@
+id=sse
```

Comment:
that was the case at the beginning but this id is used to declare the entrypoint in the api, so that was redondant and also every other plugins doesn't prefix there id by their own type (policy, connector etc)